test/python/primitives/test_backend_estimator.py

```diff
@@ -25,6 +25,7 @@
 from qiskit.providers.fake_provider import FakeNairobi, FakeNairobiV2
 from qiskit.quantum_info import SparsePauliOp
 from qiskit.test import QiskitTestCase
+from qiskit.utils.optionals import HAS_AER
```

Comment:
Is this necessary?

Reply:
It is not, I originally wrote the test using AerSimulator but then switched how the test worked

```suggestion
```